src/actionlib/action_client.py

```diff
@@ -612,6 +630,23 @@ def wait_for_server(self, timeout=rospy.Duration(0.0)):
 
         return started
 
+    ## @brief Set timestamp delta for lost goal processing
+    ##
+    ## When processing goals, compare timestamp of the goal and the
+    ## current status message. If the two differ by more than the delta,
+    ## consider the goal LOST.
+    def set_goal_ack_timeout(self, goal_ack_timeout):
+        if goal_ack_timeout:
```

Comment:
If that's the case need to do `if goal_ack_timeout is not None` otherwise it will evaluate to false if `goal_ack_timeout` is 0.